packages/backend/src/modules/dashboard/dashboard.mapper.ts

```diff
@@ -5,7 +5,15 @@ import { Mapper } from '../../application/Mapper'
 
 export class DashboardMapper implements Mapper<Dashboard, any, DashboardDTO> {
   public mapToDomain(raw: any): Dashboard {
-    return new Dashboard(raw._id, raw.title, raw.description, raw.imageCoverUrl, raw.status)
+    return new Dashboard(
+      raw._id,
+      raw.title,
+      raw.description,
+      raw.imageCoverUrl,
+      raw.createdAt,
+      raw.status,
+      raw.column
```

Comment:
Does it work? here you use ColumnDTO to app to domain type. It should be mapper from ColumnDTO to Column